db/migrate/20160713003535_add_bill_address_and_ship_address_to_customer.rb

```diff
@@ -0,0 +1,12 @@
+class AddBillAddressAndShipAddressToCustomer < ActiveRecord::Migration
+  def change
+    add_column :customers, :bill_address_id, :integer
+    add_column :customers, :ship_address_id, :integer
+
+    add_index :customers, :bill_address_id
```

Comment:
I thought it's a good practice to add index on foreign key for better performance. What do you think?